src/plugins/vis_type_timeseries/public/application/visualizations/lib/active_cursor.ts

```diff
@@ -17,9 +17,12 @@
  * under the License.
  */
 
-// TODO: Remove bus when action/triggers are available with LegacyPluginApi or metric is converted to Embeddable
-import $ from 'jquery';
+import { Subject } from 'rxjs';
 
 export const ACTIVE_CURSOR = 'ACTIVE_CURSOR';
 
-export const eventBus = $({});
+interface ActiveCursor {
+  cursor: any;
+}
+
+export const activeCursor$ = new Subject<ActiveCursor>();
```

Comment:
We are trying to avoid `any` types if possible.
Code could be also simplified, no need of usage interface.

```suggestion
import { PointerEvent } from '@elastic/charts';

export const activeCursor$ = new Subject<PointerEvent>();
```

Reply:
Thank you)